introduction.md

```diff
@@ -0,0 +1 @@
+Hello everyone. My name is Phil. And I am new to this dojo. Thank you for having me
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.